python/hdfs_downloader.py

```diff
@@ -0,0 +1,245 @@
+"""
+Reusable parallel download-to-HDFS script
+"""
+import logging
+import os.path
+
+import hdfs
+import requests
+# from requests.packages.urllib3.exceptions import InsecurePlatformWarning
```

Comment:
What do you want the final fate of `InsecurePlatformWarning` to be? I added that line because I thought it made it difficult to read the more relevant log messages, but if we do want to keep them visible we should probably just remove this entirely or make an option to turn them on and off.